views/user.ejs

```diff
@@ -20,6 +20,126 @@
+			<div class="modal-dialog">
+				<div class="modal-content">
+					<div class="modal-header">
+						<h5 class="modal-title" id="twoLabel">Zmiana Danych</h5>
+						<button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+					</div>
+					<div class="modal-body">
+						<form id="noweDane" action="/user/<%-user._id%>/newData" method="POST">
+							<div class="row">
+								<input type="text" class="form-control borderHighlight" name="newName" placeholder="Nowe imię">
+							</div>
+							<div class="row" style="margin-top: 10px">
+								<input type="text" class="form-control borderHighlight" name="NewLastname" placeholder="Nowe nazwisko">
+							</div>
+							<div class="row" style="margin-top: 10px">
+								<input type="text" class="form-control borderHighlight" name="NewUsername" placeholder="Nowa nazwa użytkownika">
+							</div>
+							<div class="row" style="margin-top: 10px">
+								<button class="btn btn-outline-secondary">Zmień</button>
+							</div>
+						</form>
+					</div>
+					<div class="modal-footer">
+						<button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Zamknij</button>
+					</div>
+				</div>
+			</div>
+		</div>
+		
+		<div class="modal fade" id="three" tabindex="-1" aria-labelledby="threeLabel" aria-hidden="true">
+			<div class="modal-dialog">
+				<div class="modal-content">
+					<div class="modal-header">
+						<h5 class="modal-title" id="threeLabel">Zmiana Hasła</h5>
+						<button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+					</div>
+					<div class="modal-body">
+						<form id="noweHaslo" action="/user/<%-user._id%>/newPassword" method="POST">
+							<div class="row">
+								<input type="text" class="form-control borderHighlight" name="oldPassword" placeholder="Podaj stare hasło" required>
+							</div>
+							<div class="row" style="margin-top: 10px">
+								<input type="text" class="form-control borderHighlight" name="newPassword" placeholder="Podaj nowe hasło" required>
+							</div>
+							<div class="row" style="margin-top: 10px">
+								<button class="btn btn-outline-secondary">Dołącz</button>
+							</div>
+						</form>
+					</div>
+					<div class="modal-footer">
+						<button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Zamknij</button>
+					</div>
+				</div>
+			</div>
+		</div>
+		
+		<div class="modal fade" id="four" tabindex="-1" aria-labelledby="fourLabel" aria-hidden="true">
+			<div class="modal-dialog">
+				<div class="modal-content">
+					<div class="modal-header">
+						<h5 class="modal-title" id="fourLabel">Zmiana Motywu</h5>
+						<button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+					</div>
+					<div class="modal-body">
+						<form id="nowyMotyw" action="/user/<%-user._id%>/newTheme" method="POST">
+							<div class="row">
```

Comment:
Każdy przycisk w oddzielnym formularzu z triggerem innego linku. Pozwoli to lepiej kontrolować przebieg.

Reply:
Nie za bardzo rozumiem. Czy już tak nie jest? Każdy formularz w ciele modala posiada osobny przycisk wywołujący odpowiednie linki.